models/demos/llama3/tt/generator_vllm.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
```

Comment:
The main reason for making it separate is that the class needs to have the decorator `@INPUT_REGISTRY.register_input_processor(input_processor_for_mllama)` and thus requires the vLLM imports. Yes it should be only used in the context of vLLM, but I'm hesitant to put it in the models folder of vLLM since it is not self-contained and still heavily relies on LlamaGenerator.

Reply:
I see. How do we handle registration in the old codebase without adding this decorator?

Reply:
We didn't need a custom input processor for llama70b (only needed for multimodal) so we never had to consider this.